workbench-core/example/app/src/routes/datasetRoutes.ts

```diff
@@ -73,11 +74,29 @@ export function setUpDSRoutes(
     })
   );
 
+  // unshare dataset
+  router.delete(
+    '/datasets/:datasetId/share/:endpointId',
+    wrapAsync(async (req: Request, res: Response) => {
+      if (
+        req.params.datasetId.match(uuidWithLowercasePrefixRegExp(dataSetPrefix)) === null ||
+        req.params.endpointId.match(uuidWithLowercasePrefixRegExp(endPointPrefix)) === null
+      ) {
```

Comment:
If either of these are null it should throw an error, not call the datasetService

Reply:
Good catch! Haven't written tests for this route yet!